hudi-common/src/main/java/org/apache/hudi/common/model/HoodieAvroPayload.java

```diff
@@ -39,11 +42,19 @@ public class HoodieAvroPayload implements HoodieRecordPayload<HoodieAvroPayload>
   private final Comparable<?> orderingVal;
 
   public HoodieAvroPayload(GenericRecord record, Comparable<?> orderingVal) {
+    this(record, orderingVal, EMPTY_PROPS);
```

Comment:
shouldn't we mark these as deprecated ?

Reply:
Based on my understanding this is only used internally and props are not used, so I don't mark it as deprecated.